builder/job.go

```diff
@@ -72,18 +73,20 @@ type Config struct {
 	cancelOnce sync.Once
 }
 
-// When called, causes the Job.WaitCancelled channel to unblock.
+// Cancel unblocks the Job.WaitCancelled channel.
```

Comment:
How about we explain what this is really for.

`Cancel signals the build job to cancel`